test/sql/function/test_suffix.cpp

```diff
@@ -0,0 +1,129 @@
+#include "catch.hpp"
+#include "test_helpers.hpp"
+
+using namespace duckdb;
+using namespace std;
+
+/* Test Case disclaimer
+ *
+ *  Assertions built using the Domain Testing technique
+ *  at: https://bbst.courses/wp-content/uploads/2018/01/Kaner-Intro-to-Domain-Testing-2018.pdf
```

Comment:
Could you run the formatter and commit again?

You can run it as follows:
`python3 scripts/format.py master`